collections/app/lib/Config.scala

```diff
@@ -0,0 +1,30 @@
+package lib
+
+import com.amazonaws.regions.{Regions, Region}
+import com.gu.mediaservice.lib.config.{Properties, CommonPlayAppConfig, CommonPlayAppProperties}
+import com.amazonaws.auth.{BasicAWSCredentials, AWSCredentials}
+
+
+object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
+
+  val appName = "collections"
+
+  val properties = Properties.fromPath("/etc/gu/collections.properties")
+
+  val awsCredentials: AWSCredentials =
+    new BasicAWSCredentials(properties("aws.id"), properties("aws.secret"))
+
+  val dynamoRegion: Region = Region.getRegion(Regions.EU_WEST_1)
+
+  val keyStoreBucket = properties("auth.keystore.bucket")
+
+  val collectionsBucket: String = properties("s3.collections.bucket")
+  val imageCollectionsTable = properties("dynamo.table.imageCollections")
```

Comment:
[We do locally](https://github.com/guardian/grid/pull/1499/files#diff-a9f99695fda2ccf6f8fd009098777753R278) (which is what we need first) - I am adding the whole stack to the CF now.